src/main/java/gregtech/loaders/recipe/handlers/PipeRecipeHandler.java

```diff
@@ -82,11 +83,11 @@ private static void processPipeTiny(OrePrefix pipePrefix, Material material, IMa
                     .duration((int) (material.getMass()))
                     .EUt(6 * getVoltageMultiplier(material))
                     .buildAndRegister();
+        } else {
```

Comment:
Don't really agree with this change. The hand crafting recipes should always be available in my opinion.